pkg/handlers/v1/testdata/ec2.deleted-malformed.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
sample message to test the ErrMissingValue is returned when ARN is not found in either spot in the config message